opensearchpy/connection/base.py

```diff
@@ -314,12 +315,17 @@ def log_request_fail(
         if response is not None:
             logger.debug("< %s", response)
 
-    def _raise_error(self, status_code, raw_data):
+    def _raise_error(self, status_code, raw_data, content_type=None):
         """Locate appropriate exception and raise it."""
         error_message = raw_data
         additional_info = None
         try:
-            if raw_data:
+            content_type = (
+                cgi.parse_header(content_type)[0]
+                if content_type
+                else "application/json"
+            )
```

Comment:
shall we pass value (eg: application/json or text ) and avoid this parser

Reply:
Replaced with something cruder (we only need the first part of the mime type)